event-listeners-mandatory-arguments/index.html

```diff
@@ -0,0 +1,18 @@
+---
+feature_name: addEventListener/removeEventListener non-optional arguments
+chrome_version: 49
+feature_id: 5640816202612736
+---
+
+<h3>Background</h3>
+<p> For a very long time, the first two arguments (<code>type</code> and
+<code>listener</code>) of <code>addEventListener</code> and
+<code>removeEventListener</code> have been optional, where they are
+non-optional in the spec and other browsers. This has changed. Now calling
+these methods with zero or one argument throw an Exception.</p>
+
+{% capture initial_output_content %}
```

Comment:
You should be able to get rid of the `initial_output_content` bit. It's optional and not used for anything here.